dashboards/nextjs/main.yml

```diff
@@ -0,0 +1,44 @@
+metric_keys:
+  - nextjs_render_time
+  - nextjs_route_change_to_render_time
```

Comment:
We usually only add one key to this because it will check all keys and that’s not necessary. I would suggest we only pick one key that we’re sure is always present in the payload the app sends.